[2025-10-31T09:55:10.488+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-10-31T09:55:10.520+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: refresh_bike_db.flask_bike scheduled__2025-10-31T09:54:00+00:00 [queued]>
[2025-10-31T09:55:10.537+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: refresh_bike_db.flask_bike scheduled__2025-10-31T09:54:00+00:00 [queued]>
[2025-10-31T09:55:10.538+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2025-10-31T09:55:10.561+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): flask_bike> on 2025-10-31 09:54:00+00:00
[2025-10-31T09:55:10.574+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'refresh_bike_db', 'flask_bike', 'scheduled__2025-10-31T09:54:00+00:00', '--job-id', '355', '--raw', '--subdir', 'DAGS_FOLDER/refresh_bike_db.py', '--cfg-path', '/tmp/tmpxuki5s2q']
[2025-10-31T09:55:10.577+0000] {standard_task_runner.py:105} INFO - Job 355: Subtask flask_bike
[2025-10-31T09:55:10.580+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=253) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-10-31T09:55:10.581+0000] {standard_task_runner.py:72} INFO - Started process 256 to run task
[2025-10-31T09:55:10.665+0000] {task_command.py:467} INFO - Running <TaskInstance: refresh_bike_db.flask_bike scheduled__2025-10-31T09:54:00+00:00 [running]> on host f4cf752b4d1e
[2025-10-31T09:55:10.830+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='refresh_bike_db' AIRFLOW_CTX_TASK_ID='flask_bike' AIRFLOW_CTX_EXECUTION_DATE='2025-10-31T09:54:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-31T09:54:00+00:00'
[2025-10-31T09:55:10.831+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-10-31T09:55:10.859+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-10-31T09:55:10.862+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '\n          set -euxo pipefail\n          cd "$AIRFLOW_HOME/plugins"\n          python app_flask.py\n        ']
[2025-10-31T09:55:10.883+0000] {subprocess.py:86} INFO - Output:
[2025-10-31T09:55:10.884+0000] {subprocess.py:93} INFO - + cd /opt/***/plugins
[2025-10-31T09:55:10.885+0000] {subprocess.py:93} INFO - + python app_flask.py
[2025-10-31T09:55:13.495+0000] {subprocess.py:93} INFO -  * Serving Flask app 'app_flask'
[2025-10-31T09:55:13.496+0000] {subprocess.py:93} INFO -  * Debug mode: on
[2025-10-31T09:55:13.521+0000] {subprocess.py:93} INFO - WARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.
[2025-10-31T09:55:13.522+0000] {subprocess.py:93} INFO -  * Running on http://127.0.0.1:5000
[2025-10-31T09:55:13.522+0000] {subprocess.py:93} INFO - Press CTRL+C to quit
[2025-10-31T09:55:13.525+0000] {subprocess.py:93} INFO -  * Restarting with stat
[2025-10-31T09:55:15.570+0000] {subprocess.py:93} INFO -  * Debugger is active!
[2025-10-31T09:55:15.580+0000] {subprocess.py:93} INFO -  * Debugger PIN: 451-352-274
[2025-10-31T10:05:41.156+0000] {local_task_job_runner.py:346} WARNING - State of this instance has been externally set to removed. Terminating instance.
[2025-10-31T10:05:41.157+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-10-31T10:05:41.160+0000] {process_utils.py:132} INFO - Sending 15 to group 256. PIDs of all processes in the group: [260, 262, 287, 256]
[2025-10-31T10:05:41.161+0000] {process_utils.py:87} INFO - Sending the signal 15 to group 256
[2025-10-31T10:05:41.162+0000] {taskinstance.py:3092} ERROR - Received SIGTERM. Terminating subprocesses.
[2025-10-31T10:05:41.163+0000] {subprocess.py:104} INFO - Sending SIGTERM signal to process group
[2025-10-31T10:05:41.197+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-10-31T10:05:41.214+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=256, status='terminated', exitcode=0, started='09:55:10') (256) terminated with exit code 0
[2025-10-31T10:05:41.724+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=287, status='terminated', started='09:55:13') (287) terminated with exit code None
